### DISCUSSION TITLE
Flatgeobufs at GopherCon 2024.

### DISCUSSION THREAD

**Discussion by bruno-ga** [flatgeobuf/flatgeobuf#324] [General]
Hello.

Just a heads up that I intend (as in, I will submit a paper proposal) to give a talk about Flatgeobufs (mostly focusing on the Go implementation) at GopherCon 2024.

@bjornharrtell do you happen to have any existing presentation about Flatgeobufs that I could use as a starting point? If not, I will create something and send to you for review if that is ok with you.

Thanks.

**Comment by bjornharrtell** [comment]
Unable to find the slides but I did do this one https://www.youtube.com/watch?v=XhEivIamRtY some time ago - perhaps some of the info in it can be of use? I think my voice sound a bit uninspired for whatever reason so I will ask you to not to use the video directly. :)

**Reply by bruno-ga** [reply]
Good stuff there, thanks. I will try to be more cheerful during my presentation (if it gets accepted). ;)
